app/models/katello/rpm.rb

```diff
@@ -215,6 +215,19 @@ def self.applicable_to_hosts(hosts)
         where("#{Katello::Host::ContentFacet.table_name}.host_id" => hosts).distinct
     end
 
+    # Return RPMs that are not installed on a host, but could be installed
+    # the word 'installable' has a different meaning here than elsewhere
+    def self.yum_installable_for_host(host)
+      env = host.lifecycle_environment
+      content_view = host.content_view
+      repos = if env.library? && content_view.default?
+                host.content_facet.bound_repositories.pluck(:id)
+              else
+                Katello::Repository.in_environment(env).in_content_views([content_view])
+              end
+      Katello::Rpm.in_repositories(repos).where.not(name: host.installed_packages.pluck(:name)).order(:name)
+    end
```

Comment:
@jlsherrill Can you tell me if I'm on the right track or off the rails here?

(This method is based off of https://github.com/jeremylenz/katello/blob/276371854df4323200b9b9414a9f7fea36fe5162/app/models/katello/concerns/pulp_database_unit.rb#L77 but I needed the same thing without the `applicable_rpms` part.)

Reply:
that looks right, but i'm not sure why you wouldn't just use host.content_facet.bound_repositories ?  That sould always point to the repos the system is using.

Reply:
> not sure why you wouldn't just use host.content_facet.bound_repositories

I didn't know what "bound repositories" means :speak_no_evil: 
```rb
repos = if env.library? && content_view.default?
                host.content_facet.bound_repositories.pluck(:id)
              else
                Katello::Repository.in_environment(env).in_content_views([content_view])
              end
```
So you're saying I don't need the `else`?

Reply:
i believe so yes.  bound_repositories are the repos reported to be in use by the client.

Reply:
The method from `pulp_database_unit.rb` that I linked above has that same `else` for non-library hosts, so that's where it came from. I'll remove it, thanks :+1: